client/packages/mobile/app/src/test/java/org/openmsupply/mobile/ExampleUnitTest.java

```diff
@@ -0,0 +1,17 @@
+package org.openmsupply.mobile;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
+    @Test
+    public void addition_isCorrect() {
+        assertEquals(4, 2 + 2);
+    }
```

Comment:
Remove?

Reply:
the whole `mobile` package was removed in next pr